Client/Frontend/Strings.swift

```diff
@@ -495,44 +431,6 @@ extension String {
             comment: "A label indicating the keyboard shortcut of shrinking the view of the current web page. This label is displayed in the Discoverability overlay when a user presses the Command key. The Discoverability overlay and shortcut become available only when a user has connected a hardware keyboard to an iPad. See https://drive.google.com/file/d/1gH3tbvDceg7yG5N67NIHS-AXgDgCzBHN/view?usp=sharing for more details.",
             lastUpdated: .v96)
 
-        public struct Sections {
-            public static let Bookmarks = MZLocalizedString(
```

Comment:
I don't see any of these used here :-| https://github.com/mozilla-mobile/firefox-ios/blob/main/Client/Frontend/Browser/BrowserViewController/BrowserViewController%2BKeyCommands.swift

Reply:
It was related to a story that turns out was never implemented https://mozilla-hub.atlassian.net/browse/FXIOS-3558

Reply:
From this group, used in #11711: Bookmarks, History, Tools, Window